--- app/Http/Controllers/RequirementController.php
+++ PHP_CodeSniffer
@@ -30,7 +30,7 @@
 
     public function storeRequirement(Request $request, $client, $project, $release, $feature)
     {
-        $requirement = New Requirement();
+        $requirement = new Requirement();
         $requirement->requirement_uuid = \Webpatser\Uuid\Uuid::generate(4);
         $requirement->feature_uuid = $feature->feature_uuid;
         $requirement->release_id = $release->release_uuid;
@@ -40,7 +40,7 @@
         $requirement->status = Status::name('draft')->id;
         $requirement->save();
 
-        if(isset($request->assignee)){
+        if (isset($request->assignee)) {
             foreach ($request->assignee as $a) {
                 $assignee = new Assignee();
                 $assignee->userid = $a;
@@ -116,7 +116,7 @@
     {
         $user = Assignee::where('uuid', $project->id)->select('userid')->distinct()->get();
         $users = User::all();
-        return view('requirement.edit_requirement', compact('client', 'project', 'release', 'feature','requirement', 'users', 'user'));
+        return view('requirement.edit_requirement', compact('client', 'project', 'release', 'feature', 'requirement', 'users', 'user'));
     }
 
     public function updateRequirement($client, $project, $release, $feature, $requirement, Request $request)
@@ -147,7 +147,7 @@
                     $assignee->save();
                 }
             }
-        } else{
+        } else {
             Assignee::where('uuid', $requirement->requirement_uuid)->delete();
         }
 
--- app/Http/Controllers/Admin/UserController.php
+++ PHP_CodeSniffer
@@ -18,10 +18,10 @@
 
     public function index()
     {
-        if (Auth::id() == 1){
+        if (Auth::id() == 1) {
             $users = User::orderBy('last_name', 'asc')->get();
 
-        return view('admin.users.index', compact('users'));
+            return view('admin.users.index', compact('users'));
         }
         abort(404);
     }
@@ -29,19 +29,20 @@
     public function dashboard()
     {
 
-        if(Auth::id() == 1){
-        $users = User::orderBy('created_at', 'desc')->get();
-        $projects = Project::orderBy('created_at', 'desc')->get();
-        $teams = Team::with('project', 'client', 'owner')->orderBy('id', 'desc')->get();
-        $plans = Plan::orderBy('id', 'desc')->get();
-        $releases = Release::get()->count();
-        $features = Feature::get()->count();
-        $requirements = Requirement::get()->count();
-
-        return view('admin.index',
-            compact('users', 'projects', 'teams', 'plans', 'releases', 'features', 'requirements'));
+        if (Auth::id() == 1) {
+            $users = User::orderBy('created_at', 'desc')->get();
+            $projects = Project::orderBy('created_at', 'desc')->get();
+            $teams = Team::with('project', 'client', 'owner')->orderBy('id', 'desc')->get();
+            $plans = Plan::orderBy('id', 'desc')->get();
+            $releases = Release::get()->count();
+            $features = Feature::get()->count();
+            $requirements = Requirement::get()->count();
+
+            return view(
+                'admin.index',
+                compact('users', 'projects', 'teams', 'plans', 'releases', 'features', 'requirements')
+            );
         }
         abort(404);
     }
-
 }

